client/src/ui/BitbakeTaskProvider.ts

```diff
@@ -108,12 +109,15 @@ class CustomBuildTaskTerminal implements vscode.Pseudoterminal {
       this.child = this.bitbakeDriver.spawnBitbakeProcess(this.command)
       this.child.stdout?.on('data', (data) => {
         this.output(data.toString())
+        logger.debug(data.toString())
```

Comment:
I think it is better to have some extra descriptive info instead of the data alone.

Reply:
I don't think so. This just pipes the bitbake terminal output into the debug log. This produces lots of text which is oviously the bitbake log. prepending every line would look bad IMO.